zkfacade/src/main/java/com/yahoo/vespa/curator/LockInfo.java

```diff
@@ -0,0 +1,64 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.curator;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Optional;
+
+/**
+ * Information about a lock.
+ *
+ * <p>Should be mutated by a single thread.  Other threads may see an inconsistent state of this instance.</p>
+ */
+public class LockInfo {
```

Comment:
Maybe `LockMetrics` is a better name for this?

Reply:
Hm, adding a LockCounters now with all the AtomicIntegers - hold on to this thought until you see that: Perhaps *that* should be called LockMetrics?  Although we are also adding metrics later, and those classes tend to be named *Metrics.